app/models/student.rb

```diff
@@ -0,0 +1,8 @@
+class Student < ActiveRecord::Base
+	belongs_to:standard
+	validates :first_name, :middle, :last_name, :date_of_birth, :address, :city, :contactno,numericality:{only_integer}, :standard_id,presence:true
```

Comment:
,numericality:{only_integer}
this is different validation, you have mentioned it like a parameter.

It should be like below.
validates :first_name, :middle, :last_name, :date_of_birth, :address, :city, :contactno, :standard_id, presence:true
validates :contactno, numericality:{only_integer},